bindings/ios/MEGASdk.h

```diff
@@ -7192,6 +7193,38 @@ typedef NS_ENUM(NSInteger, BusinessStatus) {
  */
 - (void)getRegisteredContacts:(NSArray<NSDictionary *> *)contacts delegate:(id<MEGARequestDelegate>)delegate;
 
+#pragma mark - Push Notification Settings
+
+/**
+ *@brief: Get push notification settings
+ *
+ * The associated request type with this request is MEGARequestTypeSetAttrUser
+ * - [MEGARequest paramType] - Returns the attribute type MEGAUserAttributePushSettings
+ *
+ * Valid data in the MEGARequest object received in onRequestFinish when the error code
+ * is MEGAErrorTypeApiOk:
+ * - [MegaRequest megaPushNotificationSettings] Returns settings for push notifications
+ *
+ *@param delegate MEGARequestDelegate to track this request
+ */
+- (void)getPushNotificationSettingsWithDelegate:(id<MEGARequestDelegate>)delegate;
+
+/**
+ *@brief: Set push notification settings.
+ *
+ * The associated request type with this request is MEGARequestTypeSetAttrUser
+ * - [MEGARequest paramType] - Returns the attribute type MEGAUserAttributePushSettings
+ *
+ * Valid data in the MEGARequest object received in onRequestFinish when the error code
+ * is MEGAErrorTypeApiOk:
+ * - [MegaRequest megaPushNotificationSettings] Returns settings for push notifications
```

Comment:
`MegaRequest` is not valid in the binding project

Reply:
changed the comment to MEGARequest